ansible_base/rbac/validators.py

```diff
@@ -21,7 +20,7 @@ def codenames_for_cls(cls) -> set[str]:
     return {t[0] for t in cls._meta.permissions} | {f'{act}_{cls._meta.model_name}' for act in cls._meta.default_permissions}
 
 
-def permissions_allowed_for_system_role() -> dict[Model, set[str]]:
+def permissions_allowed_for_system_role() -> dict[type, set[str]]:
```

Comment:
I don't see `type` imported, is this literally a reference to the `type()` function? It looks like from a quick google this just means some kind of complex type, is that accurate?

Reply:
Yeah, it's a python built-in, and `type()` is a callable version of `class` - for my catch-phrase description. For the RBAC model registry this comes up a lot, as I intend to return a class, similar to Django `apps.get_model()`